internal/exec/stages/disks/luks.go

```diff
@@ -41,8 +41,10 @@ var (
 
 // https://github.com/latchset/clevis/blob/master/src/pins/tang/clevis-encrypt-tang.1.adoc#config
 type Tang struct {
-	URL        string `json:"url"`
-	Thumbprint string `json:"thp,omitempty"`
+	URL                 string `json:"url"`
+	Thumbprint          string `json:"thp,omitempty"`
+	OfflineProvisioning bool   `json:"offlineProvisioning,omitempty"`
+	Advertisement       string `json:"adv,omitempty"`
```

Comment:
Does this work as a string?  The docs say "object", but they also say "raw JSON" which is contradictory.

Reply:
I presume that too.  If it works, great.